providers/tests/google/cloud/hooks/test_gcs.py

```diff
@@ -36,7 +36,7 @@
 from google.cloud.storage.retry import DEFAULT_RETRY
 
 from airflow.exceptions import AirflowException
-from airflow.providers.common.compat.assets import Asset
+from airflow.providers.google.assets.gcs import Asset
```

Comment:
I’m not sure we need to do this? I think the compat provider should always work in tests.

Reply:
Let me try it again. 🤔 If the test is using the latest released compact provider, it won't work because there's a fix in this PR. But if it's using the latest code in this PR, we should be able to revert it

Reply:
just confirmed. it works fine